components/search/src/fuse.rs

```diff
@@ -0,0 +1,52 @@
+use config::Search;
+use content::Library;
+
+/// build index in Fuse.js format.
+pub fn build_index(lang: &str, library: &Library, config: &Search) -> super::Result<String> {
+    #[derive(serde::Serialize)]
+    struct Item {
+        title: Option<String>,
+        body: Option<String>,
+        path: String,
```

Comment:
Do we really need owned values if we're just going to serialize it?

Reply:
I didn't want to mess with lifetimes and the elasticlunr logic also clones everything. But i'll change it.